template/build/webpack.prod.conf.js

```diff
@@ -120,7 +75,32 @@ const webpackConfig = merge(baseWebpackConfig, {
         ignore: ['.*']
       }
     ])
-  ]
+  ],
+  optimization: {
+    concatenateModules: true,
+    splitChunks: {
+      chunks: 'all',
+      cacheGroups: {
+        vendor: {
+          name: 'vendor',
+          test: /[\\/]node_modules[\\/]/,
+          enforce: true,
```

Comment:
I can't find any documentation on this enforce option? 
https://webpack.js.org/plugins/split-chunks-plugin/#splitchunks-cachegroups

Reply:
@zevdg Quoting @thatisuday (https://github.com/webpack/webpack/issues/6647#issuecomment-395078260):

> `enforce` will force **modules** to split whether or not condition mentioned in `splitChunks` suitable to modules to split.

If I recall correctly, I added it to ensure consistent hashing, but I really don't remember the details and, in hindsight, that probably doesn't make much sense :(. I do remember that it took me way too long to solve whatever the problem was because, as you said, there was no documentation for `enforce` so I didn't know the option existed until I came across it by accident on some unrelated article.

The [SplitChunksPlugin code](https://github.com/webpack/webpack/blob/master/lib/optimize/SplitChunksPlugin.js#L264) basically uses it to arbitrarily set other parameters to default values that guarantee that the chunk will be created.